README.md

```diff
@@ -109,8 +110,9 @@ conda activate dot
 
 Install the `torch` and `torchvision` dependencies based on the CUDA version installed on your machine:
 
+- Install cuda 11.8 from [link](https://developer.nvidia.com/cuda-11-8-0-download-archive)
 - Install `cudatoolkit` from `conda`: `conda install cudatoolkit=<cuda_version_no>` (replace `<cuda_version_no>` with the version on your machine)
-- Install `torch` and `torchvision` dependencies: `pip install torch==1.9.0+<cuda_tag> torchvision==0.10.0+<cuda_tag> -f https://download.pytorch.org/whl/torch_stable.html`, where `<cuda_tag>` is the CUDA tag defined by Pytorch. For example, `pip install torch==1.9.0+cu111 torchvision==0.10.0+cu111 -f https://download.pytorch.org/whl/torch_stable.html` for CUDA 11.1. See [here](https://pytorch.org/get-started/previous-versions/) for a list of all available `torch` and `torchvision` versions.
+- Install `torch` and `torchvision` dependencies: `pip install torch==2.0.1+<cuda_tag> torchvision==0.15.2+<cuda_tag> torchaudio==2.0.2 --index-url https://download.pytorch.org/whl/cu118`, where `<cuda_tag>` is the CUDA tag defined by Pytorch. For example, `pip install torch==2.0.1+cu118 torchvision==0.15.2+cu118 torchaudio==2.0.2 --index-url https://download.pytorch.org/whl/cu118` for CUDA 11.8.

```

Comment:
current users might face issues. why doesn't cu11.1 work?

Reply:
I updated the repo to the latest version of pytorch. In their documentation, they are using cuda 11.7 or 11.8

Reply:
if it's possible to support cu11.1 as well as cu11.7/cu11.8, that would be great.

Reply:
I tested this PR with torch1.9.0+cu111 and it seemed to be working fine.
But the old users will need to download the new checkpoints again.

Reply:
@Ghassen-Chaabouni okay. please update the README accordingly.